addon/components/ember-tbody/template.hbs

```diff
@@ -1,4 +1,5 @@
-{{#vertical-collection this.wrappedRows
+{{#vertical-collection
```

Comment:
Any reason not to convert our usage of vertical-collection to angle brackets while we're at it?

Reply:
Vertical collection is mostly using and tested with `<VerticalCollection` upstream 👍 The only reason not to use that style would be if a block has an `{{else}}`.

Reply:
We cannot quite do it yet because the `{{else}}` block is not supported in angle bracket syntax (and does not have the same equivalent.)